Add mustache comment note to FAQ (#4224)
PrairieLearn/PrairieLearn: docs/faq.md

## Why can't I connect to PrairieLearn with Docker Toolbox?

Docker Toolbox is no longer supported. [Docker Community Edition](https://www.docker.com/community-edition) is required to [run PrairieLearn locally](https://prairielearn.readthedocs.io/en/latest/installing/).

> ## How can I add a hidden Mustache staff comment in my question.html source?

Reviewer: ```suggestion
## How can I add comments in my question.html source that won't be visible to students?
```

Course staff members may want to write small maintenance comments in the `question.html` source, but HTML or JavaScript comments will remain visible in the rendered page's source (as can be seen in the browser dev tools). To prevent students from seeing staff comments, you can use [Mustache comments](https://mustache.github.io/mustache.5.html#Comments) that will be removed during the rendering process. To be safe, never put sensitive information such as solutions in a comment.

Example:

``` html

> <!-- This is an HTML comment. It will be REVEALED in the rendered page source. -->

Reviewer: ```suggestion
<!-- This is an HTML comment. It will not be visible to students in the web page, but *will be included* in the rendered page source, so students may be able to see it by reading the HTML source. -->
```

Revealed is misleading. Students still need to open the page source.

Author: Would it suffice to just put the second half? That is, "This is an HTML comment. Students may be able to see it by reading the rendered HTML source."

Reviewer: Yeah, I guess so...

Author: I'll just commit your suggestion as-is. I wasn't sure if I should put any explanation inside the example comments or not for brevity.
